nethcti-server/Containerfile

```
@@ -1,28 +1,43 @@
+# clone repository using alpine container
+FROM docker.io/library/alpine:3.17.2 as repofetcher
+WORKDIR /app
+RUN apk add --no-cache git
+ARG NETHVOICE_SERVER_COMMIT=97ae4c9b054bdc289d660ea49da538c7862d2ac1
+RUN git clone https://github.com/nethesis/nethcti-server.git . \
+    && git checkout "${NETHVOICE_SERVER_COMMIT}"
```

I think this is the correct approach: you know exactly what's inside the image.
Still, I think in this development phase we are going to make many changes and we could forget to bump the hash.

What about, for now, just cloning the ns8 branch?
```suggestion
RUN git clone -b ns8 --depth=1 https://github.com/nethesis/nethcti-server.git .
```

Still, feel free to keep the current implementation.

---

Ok, let's keep it.
If we face development issues we will change it a bit.